medusa/providers/nzb/omgwtfnzbs.py

```diff
@@ -86,7 +86,7 @@ def search(self, search_strings, age=0, ep_obj=None):
 
                 for item in response:
                     try:
-                        if not self._get_title_and_url(item):
+                        if not all(self._get_title_and_url(item)):
```

Comment:
@medariox need to do this to all providers as the old check return true even if one of items in tuple are none